docs/src/main/sphinx/connector/hive.md

```diff
@@ -1069,6 +1069,12 @@ In Trino, these views are presented as regular, read-only tables.
 Hive views are defined in HiveQL and stored in the Hive Metastore Service. They
 are analyzed to allow read access to the data.
 
+You should migrate any existing Hive views that you want to access from Trino to
```

Comment:
I don't quite agree with this statement.
I'd recommend removing it.

Reply:
done